### ISSUE TITLE
Bug Report (IF I DO NOT CHANGE THIS THE ISSUE WILL BE AUTO-CLOSED)

### ISSUE THREAD

**Issue by kshji** [microsoft/terminal#2253]
# Environment

```none
Windows build number: [run "ver" at a command prompt]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

# Expected behavior

# Actual behavior

**Comment by kshji** [comment]
issue https://github.com/microsoft/terminal/issues/521 is same problem. Default mapping is not for international using.

**Comment by zadjii-msft** [comment]
Yea this is a dupe of the discussion happening in #521 currently. It'll be fixed by #2235
